.travis.yml

```diff
@@ -1,27 +1,33 @@
+    # Since we're using conventional commit paradigm, we only want a master merge commit to trigger a build: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
+    - master
 env:
   global:
-    - secure: MEL/fse4IwgsR+Kxws8znyk4Zz+F/pBWibrOBV93xSL7oImXRPIildatX3kpk0nIDHdDreWlzeXHQzmse6S50HpFvseZd0XWWVbQGIQVRda13R3zYbsHluI/Y5Rsh5UZysd3KV0MXyfbvR7xMNF70/IasmOCYtncTxztL5Garrn9jDlNnbkSj7HCbozv+IaarL9q9GOwHO5uWbDvOQLxax9VOOF50A34EBkmrV3xKpU2Iaa5+MZBWLIQ0tov5lzEcBN2fLKoblko96JMd2jk3dQQ+CLkfZPG3LaKc+XYsmeGa4ZhWpOTnbukT43I+ZmKu0rGyqvUvMYPPJElq9ttINloxkgDFU6Qkgb8je6U1kVQ8Tj9ZeN5JiMfEHthvXeCRmZFlgk1i94XyF16L5QScpXBZu0olaobvm3QDhITOzeH8ytcZ/Pgwr1cHwcjujiNK/1qQZuZGdUk6heuCHwt7FBocH66Xugf1BEs67F0H1btvGDXkAbAnMpO1/HvrQlnQYhi/m0fACALAPEWRyOe3rAVWsM70/36BhMQ4ylZYl/U8H/7MCw+XZh4MwrYByCLtvm0fTPPHpq9VyuoLlU7vHvsa7Any5gNB8qsWDdhfzZdqYstH438hd7yYIMNT1AH4UgHEi8bk+Scp8Jht9ZiI230i/VGOwAdEMskUvv5kPc=
-    - secure: YOOuOXd7osj7GE2RvK8WZhF5eobJzrgjVnf1jSI6wgVYB8RwnN+EP1+aKfqAglDtou5q3I80QlKAln2x6QomIrdJW1GF5elVGFBxBO1L/Xk/jb3D9an9GLPHKZyO+hfJo8HDzkLaiBHNI8YYeqrEXQU4wpzv5uaNT7Rer1Dyy4DuTdZ/N4iFmX8w36kPoPXcSWyu71hNZ54btnb7cvl4XkwX5/qAXyIarGzRK+x0V6rAM1PC1fcam1udivsN1py+PVuxbgv3VCt/6kd2HrJ88y9/ZOIjE35av9/b0EEkVJb7LX/FPxpiXVjyKtDgQtdGvLKrcbL+6KZrxCv71Qbkrd36kR19LTLPFuy40y2sL+NN4bQWkI+dFCg/tQyQWNFNpnuTw/+g8/kHaMqutyFIIN79j1KGRY0wm6eVG6E4YA+hUuAkdXdaN0U9cD89Hf+mIn+EGEWTySwLcH3wWxkqtqAn6D3n81J4kfsv1jObuPtbHbM5ONCbkSCLqvW/djx1PblVT5sSLbnpXGfp4KlEYbTTJ/fyp7QR/68RftwadgipAPMJDvWxZISMZxrgHTg+bhQgByGR0rpJl1/BDo3OLbW27B7WW3dn/FBZF8a4amq1zn2Lg34w+M5xjRqOOUv44QToN5CC3tWoVgccPjbq3sdEpWYGa48j8Vh6LHGc1zM=
-    - secure: FPmvf0t+h0/RE+10bERX8gatJpxSbf62T3LTH8omy6WRWlPZL9VgO0dfX0ZZzmkfAHH//zTB5YEdwE7ntwdzWaIx1ti7RjanRlznlrOLOwP55CDIu9/nvl1spMC4vP2sGz6l7iHJCbiJfXYT5xfkk6DezaWijsolsE6DWoFXge/Q3Q2BkPrlUoBcdZvl8cPQ+Mv003XSzX+3GdCrCkz9Eb4j16goFLLlAkQEFlFgTOVYv4JtcCgVxBzSDqmvITKCLQXJ3QlSHy1ZtRYnZCt2qO8UMJZQA7t7hGXrlcR59arboWUiMfNRZ3A3N5kKLyDUi+4AaaxglCwkSBFwRSuMy8rEul0VFPob0iOHt42rYGmcqGS8CTaRwZFqgJEZBIGgjLbh7onXaRTPlIYPz/+dHs3Msi56lO7ZBqHdxtxlgRL5mC/ENAnCk6WLYgXWOO1MlLFWy0KKlYe/weoSUbBgPux58zQl7sglVE2B37Se+c4gjW/VC3+kmw36j0nr5GedFUZse4rtkPt+OP8aCUo+lTUgBGBbOQgeipWGJNFm9hnTWSyod/GUeF3bUJLUPkx95hF+8/apmKONuXKG9X6XJ7aeCzVo1pmAwJ4hAw378AXdscuaTfZaxsHy4IdqbrnMdPR4p50rZFGk3ojYQR/HQKwWajlYX8mUuDToVLrSzgg=
-    - secure: hw9IKh419UicjaMzoJuD/Lu8zaXBqHl2B277pc27ja+GaicZ6ruttQajV8Vxnqre+lRGDp8TPnTiGbuRSjB+UEAiVHGXF8ox5Ewsu33HC40YtBBudJZPAfdSijC1yA+tBAw3cuR3NDnB4mDu/t6/LyXmBZCanTZVF2Lw4vtCKxjfHndVZUadtml/d2+Ym98daKmZUI1Qnqnrkp5N5qlqPpE/LlH7SvxlcG37KwoFy16TKYYpMxt9GJ3T/vy4eytcwXUawYJBMnJA7uyiC4J9gWUluGk0EByuCzs/wuKZP+tXhHxdjfPWpFVf6yuwHS10DZQbqSAs5DVqb0nsM39BKnL6A5Vkq7yuB+qAeaWSDu1w2KPpyxfnlwYCsfq+Kf7U0AM43QSw1GEfeR949YMLjGfDTO/DOZivqzkZvBdREiNFf8AHPdWFYsgZPBGhVeQ4uQbWc3kLlcgNzjsyTSns8aqB5sEy238JgZrMke0mB6UMQNa14cmiypCnB94VsBED8QvVCbSvN3C29cQPcVUlb5QJdYPDXPLXWT64VZMlDypVifx9MC44FMYScqfEF35x4btitTJDdFx70f9RNdLPiac/wwl/5TBBmMDWU84LEqBIkdU+Rf2ubc9YLux+OIfnA+TE9X1Ynm/iyEq/su6RQt7RlKKDmRUJRywiMs0l+Pk=
```

Comment:
This part looks like a typo: "adtml/d2+Ym98"

Just kidding.

Reply:
:P. We should meet up soon and I can run through all of the travis things with you. It's pretty cool.